src/src/utils/auth.js

```diff
@@ -0,0 +1,19 @@
+import Cookies from 'js-cookies';
+import decode from 'jwt-decode';
+
+const Tokenkey = 'vue_admin_identity_token';
+const StateKey = 'vue_admin_identity_state';
+const TenantApplicationIdKey = 'tenant_application_id';
+
+export function getSolutionId(){
+    return;
+}
+export function getUsername(){
+    return Cookies.get('vue-username');
+}
+export function getLanguageCode(){
+    return Cookies.get('language');
+}
+export function getUserId(){
+    return Cookies.get('vue-userid');
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  return Cookies.get('vue-userid');
```